### ISSUE TITLE
nexmo core update

### ISSUE THREAD

**Issue by Francois-Francois** [laravel/nexmo-notification-channel#60]
- Vonage Notification Channel Version: 3.0.0
- Laravel Version: 9.9.0
- PHP Version: 8.1.4
- Database Driver & Version:

### Description:

Due to this commit https://github.com/Vonage/vonage-php-sdk-core/commit/8ffaa3e8f589e4c8946bf972558283cf584bc360 when sending sms, it throw a RuntimeException : ```no api namespace found: message```

This is because `message()` method is not supported anymore. Looks like it has been replace by `sms()` but it signature is not the same because when forcing sms() it throw another error : 
```
Vonage\SMS\Client::send(): Argument #1 ($message) must be of type Vonage\SMS\Message\Message, array given, called in /laravel/vonage-notification-channel/src/Channels/VonageSmsChannel.php on line 73
```

### Steps To Reproduce:

Just by trying to use 'vonage' notification channel : 

```php
public function toVonage(User $notifiable)
{
        return (new VonageMessage)
            ->content('content')
            ->unicode();
}
```

**Comment by driesvints** [comment]
Duplicate of https://github.com/laravel/vonage-notification-channel/issues/59

There's nothing we can do about this.

**Comment by Francois-Francois** [comment]
Sorry for the duplicate. Didn't see it.

But I think this should be a good idea to switch on the newest api method : sms() and upgrade the channel.

Or maybe I'm missing something @driesvints ?

**Comment by driesvints** [comment]
Last time I checked it wasn't easy to do that. We're welcoming PR's however if you can figure out a way without breaking changes.

**Comment by Francois-Francois** [comment]
The temp fix for me is to force "vonage/client-core": "3.0.2"

3.0.3 introduces the bug

I'll look for a PR, but I think it's something we have to consider, because they "removed dead code" if we look at the commit, so all vonage notifications will be in error for laravel users.

**Comment by driesvints** [comment]
The Vonage team is aware of it and are going to address it as soon as they can. Thanks

**Comment by ShapesGraphicStudio** [comment]
Hi, looks like this is still not fixed in v3.2.1.. ?
